omero/developers/json-api.rst

```diff
@@ -210,7 +210,7 @@ It uses the python ``requests`` library to connect to the JSON api, login, query
 These steps are covered in more detail below.
 
 For an example how to use the API with Java,
-see :omedocs:`blob/develop/omero/examples/java/JSONClient.java`.
+see :omedocs:`JSONClient.java <blob/develop/omero/examples/java/JSONClient.java>`.
```

Comment:
Someway I had missed we are now including example files in this repository. In this case, instead of hardcoding the GitHub URL and dealing with branch, it might be simpler to use ```:download:`JSONClient.java </examples/java/JSONClient.java>` ``` as we do for the Bio-Formats examples for instance.